alf/bin/train.py

```diff
@@ -91,6 +94,14 @@ def main(_):
     root_dir = os.path.expanduser(FLAGS.root_dir)
     os.makedirs(root_dir, exist_ok=True)
     logging.get_absl_handler().use_absl_log_file(log_dir=root_dir)
+
+    if FLAGS.store_snapshot:
+        # ../<ALF_REPO>/alf/bin/train.py
+        common.set_alf_root(
+            str(pathlib.Path(__file__).parent.parent.parent.absolute()))
```

Comment:
if _alf_root isn't used elsewhere, then just remove this global variable and instead pass it `str(pathlib.Path(__file__).parent.parent.parent.absolute())` in as a second argument to the following generate_alf_root_snapshot function?

Reply:
I just added _alf_root for potential future use.

Reply:
Actually I decided to remove _alf_root global variable.